hazelcast/include/hazelcast/client/HazelcastClient.h

```diff
@@ -503,6 +504,13 @@ namespace hazelcast {
                 return IMap<K, V>(proxy);
             }
 
+            map::MixedMapProxy getMixedMap(const std::string &name) {
```

Comment:
I think API name should be `MixedMap` rather than `MixedMapProxy`

Reply:
May be we can use `adaptor`  approach as we do RawPointerMap ? 
Is there a specific reason why we choose a different approach ?

Reply:
Adaptor could not apply in this concept, because what scan we adopt? IMap requires type information. Yes, I will put MixedMap instead of MixedMapProxy for the user API.